# Message describing single TopoNavEdge

int64 edge_id
time last_updated
int64 start_node_id
int64 end_node_id
float64 cost
